docs/guides/basic-rbac.mdx

```diff
@@ -3,267 +3,303 @@ title: Implement basic Role Based Access Control (RBAC) with metadata
-
-```tsx {{ filename: 'src/app/admin/dashboard/page.tsx', mark: [[3, 5], 7, 12, 14, 21, [23, 52]] }}
-import { redirect } from 'next/navigation'
-import { checkRole } from '@/utils/roles'
-import { SearchUsers } from './_search-users'
-import { clerkClient } from '@clerk/nextjs/server'
-import { setRole } from './_actions'
-
-export default async function AdminDashboard(params: { searchParams: { search?: string } }) {
-  if (!checkRole('admin')) {
-    redirect('/')
-  }
+  export default async function AdminDashboard(params: { searchParams: { search?: string } }) {
+    if (!checkRole('admin')) {
+      redirect('/')
+    }
 
-  const query = params.searchParams.search
+    const query = params.searchParams.search
 
-  const users = query ? (await clerkClient().users.getUserList({ query })).data : []
+    const users = query ? (await clerkClient().users.getUserList({ query })).data : []
 
-  return (
-    <>
-      <h1>This is the admin dashboard</h1>
-      <p>This page is restricted to users with the 'admin' role.</p>
+    return (
+      <>
+        <p>This is the protected admin dashboard restricted to users with the `admin` role.</p>
 
-      <SearchUsers />
+        <SearchUsers />
+
+        {users.map((user) => {
+          return (
+            <div key={user.id}>
+              <div>
+                {user.firstName} {user.lastName}
+              </div>
+
+              <div>
+                {
+                  user.emailAddresses.find((email) => email.id === user.primaryEmailAddressId)
+                    ?.emailAddress
+                }
+              </div>
+
+              <div>{user.publicMetadata.role as string}</div>
 
-      {users.map((user) => {
-        return (
-          <div key={user.id}>
-            <div>
-              {user.firstName} {user.lastName}
-            </div>
-            <div>
-              {
-                user.emailAddresses.find((email) => email.id === user.primaryEmailAddressId)
-                  ?.emailAddress
-              }
-            </div>
-            <div>{user.publicMetadata.role as string}</div>
-            <div>
               <form action={setRole}>
                 <input type="hidden" value={user.id} name="id" />
                 <input type="hidden" value="admin" name="role" />
                 <button type="submit">Make Admin</button>
               </form>
-            </div>
-            <div>
+
               <form action={setRole}>
                 <input type="hidden" value={user.id} name="id" />
                 <input type="hidden" value="moderator" name="role" />
                 <button type="submit">Make Moderator</button>
               </form>
             </div>
-          </div>
-        )
-      })}
-    </>
-  )
-}
-```
+          )
+        })}
+      </>
+    )
+  }
+  ```
 
-## Finished 🎉
+  ### Finished!
```

Comment:
I miss the 🎉 (not a plea to add it, just a silly comment)

Reply:
[dd80c54](https://github.com/clerk/clerk-docs/pull/1560/commits/dd80c5481f292b5631858661dd17fd431588d03e)

hehe